### ISSUE TITLE
torchscript_traced_mask_rcnn unexpected masks

### ISSUE THREAD

**Issue by marcobevih2o** [facebookresearch/detectron2#3662]
i follow instructions to compile and test torchscript_traced_mask. all went fine, i got outputs with correct detection class and BB, but masks for all inputs are 1x28x28. why? i tried with coco dataset, with my own training set and size mask are always the same

**Comment by marcobevih2o** [comment]
My environment:

os: Kubuntu 18.04
cuda version: 11.2
cudnn version: 8.1
torch version: 1.11.0a0+git14ee608 (compiled from source)
torchvision version: 0.12.0a0+a3541eb
detectron2 version: 0.5

**Comment by ppwwyyxx** [comment]
This is expected and documented. https://github.com/facebookresearch/detectron2/discussions/3393 has more explanations.

I might add a version later that produces full-image-size masks.
